core/service/src/consts.rs

```diff
@@ -155,6 +155,13 @@ lazy_static::lazy_static! {
         1u8, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 3,
         4,
     ]);
+
+    // The default epoch ID used for initial PRSS setup and as fallback when no epoch is specified.
+    // This is equivalent to PRSS_INIT_REQ_ID parsed as an EpochId.
+    pub static ref DEFAULT_EPOCH_ID: EpochId = EpochId::from_bytes([
+        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
Maybe we should actually use a new byte prefix for this since we will now support new epoch requests?  But I guess this should be coordinated with gateway and is not important now. Just want to ensure it is something we keep in mind